pkg/api/handlers/libpod/pods.go

```diff
@@ -530,8 +530,12 @@ func PodStats(w http.ResponseWriter, r *http.Request) {
 	query := struct {
 		NamesOrIDs []string `schema:"namesOrIDs"`
 		All        bool     `schema:"all"`
+		Stream     bool     `schema:"stream"`
+		Delay      int      `schema:"delay"`
 	}{
 		// default would go here
+		Delay:  5,
+		Stream: true,
```

Comment:
I'd argue that the default should remain false so we don't break compatibility with older versions

Reply:
I can change that to false, but it's making a feature out of a bug :) According to the docs it should be streaming from the beginning.